node_generator/node_generator.cc

```diff
@@ -461,6 +462,52 @@ void NodeGenerator::CleanupAndExit(int exit_code) {
     exit(exit_code);
 }
 
+std::string NodeGenerator::get_binary_path() const {
+    std::filesystem::path wrapper_script_path = std::filesystem::path(repo_root_) / "bazel-bin" / "ros2";
+    return wrapper_script_path.string();
+}
+
+std::string NodeGenerator::DetermineRepoRoot() const {
```

Comment:
@ungheele I am currently trying to remove RepoRoot and make this run as a binary. Let's get my PR merge and closed this one.